openehr-rm/src/main/java/com/nedap/archie/rm/archetyped/FeederAuditDetails.java

```diff
@@ -32,17 +33,21 @@
     @Nullable
     @XmlElement(name = "version_id")
     protected String versionId;
+    @Nullable
+    @XmlElement(name = "other_details")
+    private ItemStructure otherDetails;
 
     public FeederAuditDetails() {
     }
 
-    public FeederAuditDetails(String systemId, @Nullable PartyIdentified provider, @Nullable PartyIdentified location, @Nullable DvDateTime time, @Nullable PartyProxy subject, @Nullable String versionId) {
+    public FeederAuditDetails(String systemId, @Nullable PartyIdentified provider, @Nullable PartyIdentified location, @Nullable DvDateTime time, @Nullable PartyProxy subject, @Nullable String versionId, ItemStructure otherDetails) {
```

Comment:
These constructor updates make the changes backwards incompatible. Is that OK?

Reply:
Good question! Not sure. I'll ask the people from EHRBase, I think they are the main users, and they have experience on using archie as a drop-in replacement from the older java implementation.